CONTRIBUTING.md

```diff
@@ -0,0 +1,82 @@
+2. Create a new development branch off of the upstream (also known as the original repository's) `integration` branch.
+   - `git checkout -b <my-new-feature-branch> integration`
+3. Do your work, including writing code, writing tests, updating documentation, and passing all tests locally.
+4. Fetch the latest upstream changes to ensure you have the latest working copy of the upstream codebase.
+   - `git fetch upstream`
+5. Rebase your changes on top of the latest upstream streams, being sure to resolve any conflicts. _(This process allows you to resolve any local issues with your changes, one by one, before submitting your PR instead of afterwards.)_
+   - `git branch --set-upstream-to=upstream/integration`
+   - `git rebase`
+6. Push your rebased change set to your repository.
+   - `git push origin <my-new-feature-branch>`
+7. Create a pull request from your `<my-new-feature-branch>` against the upstream **`integration`** branch.
+   1. _**NOTE:**_ The target branch of your Pull Request should be `integration`, not `main`!
+   2. Either in the initial comment section of your Pull Request or in the **Reviewers** section, you will need to tag **@cortx-k8s-admins** for the project maintainers to be made aware of your pull request.
+   - _Reference:_ [Creating a pull request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request)
+8. Your PR will be reviewed, inspected, and (if appropriate) automatically tested using integrated CI/CD workflows. Once complete, project maintainers will accept the PR and merge the code changes into the `integration` branch.
+9. Congratulations! You have now successfully contributed to making the CORTX project even better!
+
+## Additional Resources
+
+- Learn more about the [CORTX Architechture](https://github.com/Seagate/cortx/blob/main/doc/architecture.md). 
+- Learn more about [CORTX CI/CD and Automation](https://github.com/Seagate/cortx/blob/main/doc/CI_CD.md).
+- Browse our [suggested list of contributions](https://github.com/Seagate/cortx/blob/main/doc/SuggestedContributions.md).
+
+## Communication Channels
+
+Please refer to the [Support](https://github.com/Seagate/cortx/blob/main/SUPPORT.md) section in the root [CORTX](https://github.com/Seagate/cortx) repository to learn more about the various channels by which you can reach out to us. 
+
+## Maintainers
+
+Active and past maintainers of the [cortx-k8s](https://github.com/Seagate/cortx-k8s) project are tracked via the [MAINTAINERS.md](MAINTAINERS.md) page. Please use this page for further discussion, communication, and questions as needed while working with CORTX on Kubernetes.
+
+## Acknowledgements
+
+After making a contribution, please don't forget to include yourself in our [Contributors list](https://github.com/Seagate/cortx/blob/main/CONTRIBUTORS.md) in the root [CORTX](https://github.com/Seagate/cortx) repository!
```

Comment:
I notice that there is no "cortx-k8s" section in the cortx contributors list.  Is this just because no active contributors have bothered to create that section?  (Or is it a problem with this doc?)

Reply:
Good catch. I think we just forgot to do so. I just added it.